code_generation/analysis_template.cxx

```diff
@@ -108,6 +110,8 @@ int main(int argc, char *argv[]) {
 
     // start an onnx session manager
     OnnxSessionManager onnxSessionManager;
+    // start a correction manager
+    CorrectionManager correctionManager;
```

Comment:
I think we should also add this in `analysis_template_friends.cxx`. For example for fake factors also a correctionlib file is used.